suite/tests/CMakeLists.txt

```diff
@@ -3657,7 +3657,7 @@ if (BUILD_CLIENTS)
     set(tool.drcov.fib_expectbase "tool.drcov.fib")
     DynamoRIO_get_full_path(tool.drcov.fib_postcmd drcov2lcov "${location_suffix}")
 
-    if (UNIX AND NOT RISCV64) # TODO i#3544: Port tests to RISC-V 64
+    if (UNIX)
       # Test an app that executes a pipe syscall for i#5981.
       torunonly_ci(tool.drcov.eintr linux.eintr drcov linux/eintr.c "" "" "")
```

Comment:
Add a `TODO #3544` comment about adding this to the RUNS_ON_QEMU list, explaining it's a QEMU issue that's preventing it.  Maybe better to put that comment down by the RUNS_ON_QEMU list itself.  Let's not have future readers think you made a mistake and forgot to add it when you enabled the build: use comments to make an omission like this clear.

Reply:
https://github.com/DynamoRIO/dynamorio/blob/a2f8bcc95def89de92141e627e53c796b1377089/suite/tests/CMakeLists.txt#L6300
I see. But there's already such a comment before the RUNS_ON_QEMU list.

Reply:
That doesn't imply that those tests succeed by themselves on bare metal: it just says they build.